ports/mimxrt/mpconfigport.h

```diff
@@ -145,6 +145,7 @@ __attribute__((always_inline)) static inline void enable_irq(uint32_t state) {
 
 __attribute__((always_inline)) static inline uint32_t disable_irq(void) {
     uint32_t state = __get_PRIMASK();
+    __set_PRIMASK(1);
```

Comment:
does it work with `__disable_irq()` here instead, like is done in stm32?

Reply:
Yes. I used that at another place too. But there is also the NXP-Lib Macro pair of DisableGlobalIRQ() and EnableGlobalIRQ(), which is effectively the same. It is defined as:
```
    uint32_t regPrimask = __get_PRIMASK();
    __disable_irq();
    return regPrimask;
```
So I tend to use that one.

Reply:
No. I'll stick with __disable_irq().